pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -23,6 +23,21 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// DefaultPreemptionArgs holds arguments used to configure the
+// DefaultPreemption plugin.
+type DefaultPreemptionArgs struct {
+	metav1.TypeMeta
+
+	// MaxCandidatesAsPercentageOfNodes is the maximum number of candidates to
+	// shortlist when dry running preemption as a percentage of number of nodes.
+	MaxCandidatesAsPercentageOfNodes int32
```

Comment:
I feel the name can be simplified, like `PercentageOfCandidateNodes`?

Reply:
or `MaxCandidateNodesPercentage`. I think it's a good practice to have the "units" at the end of the name.

Reply:
Done.